runtime/compiler/build/files/common.mk

```diff
@@ -380,8 +380,19 @@ JIT_PRODUCT_SOURCE_FILES+=\
     omr/compiler/runtime/OMRCodeCacheMemorySegment.cpp \
     omr/compiler/runtime/OMRRuntimeAssumptions.cpp
 
+ifneq ($(JITSERVER_SUPPORT),)
```

Comment:
Please don't change the permissions on this file: execute permissions are not needed.